src/controllers/best-sellers.js

```diff
@@ -0,0 +1,154 @@
+import { i19sku, i19name, i19price, i19sales, i19total } from '@ecomplus/i18n'
+import { $ecomConfig, i18n, formatMoney } from '@ecomplus/utils'
+import Papa from 'papaparse'
+
+export default function () {
+  const { $, callApi, tabId } = window
+
+  const datatableOptions = {
+    pageLength: 20,
+    bLengthChange: false,
+    order: [[3, 'desc']]
+  }
+  if ($ecomConfig.get('lang') === 'pt_br') {
+    datatableOptions.language = {
+      aria: {
+        sortAscending: ': ative para colocar a coluna em ordem crescente',
+        sortDescending: ': ative para colocar a coluna em ordem decrescente'
+      },
+      paginate: {
+        next: 'Próxima',
+        previous: 'Anterior'
+      },
+      emptyTable: 'Tabela vazia',
+      info: 'Mostrando _START_ a _END_ de _TOTAL_ SKUs carregados',
+      infoEmpty: '',
+      infoFiltered: '',
+      lengthMenu: 'Mostrar _MENU_ resultados',
+      search: 'Buscar',
+      zeroRecords: 'Nenhum resultado encontrado'
+    }
+  }
+
+  const datatable = $('#most-sellers-table').DataTable(datatableOptions)
+  const renderGraphByDate = (start, end) => {
+    callApi(
+      '$aggregate.json',
+      'POST',
+      (err, json) => {
+        if (!err) {
+          const { result } = json
+          const rows = []
+          if (Array.isArray(result) && result.length) {
+            result.forEach(item => {
+              if (item.summary && Array.isArray(item.summary) && item.summary.length) {
+                let summary = item.summary[item.summary.length - 1]
+                rows.push([
+                  item._id,
+                  summary.name,
+                  formatMoney(item.amount / (item.count || 1)),
+                  item.count || 0,
+                  formatMoney(item.amount)
+                ])
+              }
+            })
+            datatable.clear()
+            datatable.rows.add(rows)
+            datatable.draw()
+            $('#most-sellers-table-load').hide()
+          }
+        }
+      },
+      {
+        resource: 'orders',
+        pipeline: [
+          {
+            $match: {
+              created_at: {
+                $gte: `${start}T03:00:00.000Z`,
+                $lte: `${end}T02:59:59.999Z`,
+              },
+              'financial_status.current': 'paid',
+            }
+          },
+          {
+            $unwind: '$items'
+          },
+          {
+            $group: {
+              _id: '$items.sku',
+              count: {
+                $sum: '$items.quantity'
+              },
+              amount: {
+                $sum: {
+                  $multiply: ['$items.price', '$items.quantity']
+                }
+              },
+              summary: {
+                $addToSet: {
+                  sku: '$items.sku',
+                  name: '$items.name',
+                  price: '$items.price'
+                }
```

Comment:
```suggestion
                sku: { $first: '$items.sku' },
                name: { $first:  '$items.name' }
```

Você já não usa `price` aqui (nem deveria, pega o preço médio pelo montante / quantidade), fora isso `$addToSet` não faz sentido, operação mais lenta e output é mais complexo do que você precisa, já que só vai usar o primeiro nome e SKU de qualquer maneira...

Reply:
> `summary` deixa de ser um array no seu output, sei trecho que trata ele precisa ser alterado também